es/common_voice_terms.md

```diff
@@ -1,67 +1,79 @@
 # Términos legales de Common Voice
 
-Vigentes desde el 30 de noviembre de 2018 {: datetime="2018-11-30" }
+En vigor desde el día jueves, 27 de junio de 2024 {: datetime="2024-06-27" }
 
 Colabore con nosotros a través de Common Voice donando su voz, frases escritas y otros recursos para construir una base de datos de voz de código abierto disponible para cualquier persona que desee crear aplicaciones innovadoras de reconocimiento de voz para dispositivos y la web.
 
-Solo puede participar en Common Voice si está de acuerdo con estos Términos.
+Solo puedes participar en Common Voice si aceptas estos Términos Legales de Common Voice (los "Términos"). 
 
 ## 1. Eligibilidad
+
 Para colaborar en Common Voice debe tener más de 19 años de edad. Si tiene 19 años o menos, podrá participar con el consentimiento y bajo la supervisión de su padre, madre o tutor legal durante su participación en Common Voice.
 
 Common Voice forma parte de la comunidad de Mozilla. Es decir, que si decide participar, apepta adherirse a las [Pautas para la participación en la comunidad de Mozilla](https://www.mozilla.org/about/governance/policies/participation/). 
 
-## 2. Sus contribuciones 
-La base de datos de Common Voice de Mozilla está disponible en [Ofrecimiento al dominio público de Creative Commons CC0](https://creativecommons.org/publicdomain/zero/1.0/). Esto significa que es pública y que renunciamos a los derechos de autor en la medida que lo permite la ley. Si participa en Common Voice, requerimos que usted haga lo mismo. Debe permitir que Mozilla pueda compartir sus contribuciones (por ejemplo, textos, grabaciones, validaciones y comentarios) para su uso en proyectos de dominio público [CC0](https://creativecommons.org/publicdomain/zero/1.0/). 
+## 2. Tus contribuciones
+
+Ponemos la base de datos de Common Voice de Mozilla a disposición bajo la [dedicación de dominio público Creative Commons CC0](https://creativecommons.org/publicdomain/zero/1.0/). Eso significa que es pública y hemos renunciado a todos los derechos de autor en la medida en que podamos bajo la ley. Si participas en Common Voice, requerimos que hagas lo mismo. Aceptas que Mozilla puede ofrecer al público todas tus contribuciones a Common Voice, incluyendo texto, grabaciones, validaciones y comentarios (las "Contribuciones") bajo la dedicación de dominio público de [CC0](https://creativecommons.org/publicdomain/zero/1.0/). 
```

Comment:
> Aceptas que Mozilla puede ofrecer al público todas tus contribuciones a Common Voice,

Will follow up via email, but it doesn't look like Spanish took in the last update.